vajram/src/main/java/com/flipkart/krystal/vajram/inputs/From.java

```diff
@@ -4,6 +4,6 @@
 import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
-public @interface BindFrom {
+public @interface From {
```

Comment:
`ResolveDep` will go better with `Using` instead of `From`-> It will read like this:
ResolveDep, name, inputs, using(a), using(b) etc.

For example:
```
@ResolveDep(depName="dependency", depInputs="input")
static compute(@Using("a") int a, @Using("b") int b){
  return a+b;
}
```

@prateek-kr-fk What do you think?

Reply:
Changed to Using